### PR TITLE
add migration companies table and seeds

### PR DESCRIPTION
**Pull request by claudion3**
# Description

This PR is about Create a Knex migration for the Companies table and  adding a seed file for development with  couple of initial companies

Fixes 25

# How to test?

I run in MySQL workbench and is working 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by Deepti-Dk — COMMENTED** [review]
Code looks 👍 just a little thing and its not about code. Under 'description' where it says Fixes... if you remove the space after #, it will become a link (easy to track :-)

File: src/server/migrations/20201119023251_create_company_table.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+  // Create company table
+  return knex.schema.createTable('company', (table) => {
+    table.increments();
+    table.string('name').notNullable();
+    table.string('menton').notNullable();
```
[the remark this answers is not in the document]
Comment (claudion3) [reply src/server/migrations/20201119023251_create_company_table.js:6]: yes i will correct it, thx

[the remark this answers is not in the document]
Comment (Elia-Git) [reply src/server/migrations/20201119023251_create_company_table.js]: Good job!

**Review by Elia-Git — APPROVED** [review]

**Comment by dpfernandes** [comment]
Good job but we will handle companies as organisations and we will do it through this PR https://github.com/HackYourFuture-CPH/web-dev-rooms/pull/73

**Pull request closed without merging**